bazel/repository_locations.bzl

```diff
@@ -741,11 +741,11 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "Webassembly Micro Runtime",
         project_desc = "A standalone runtime with a small footprint for WebAssembly",
         project_url = "https://github.com/bytecodealliance/wasm-micro-runtime",
-        version = "WAMR-08-10-2021",
-        sha256 = "4016f8330b2ed4fb5d9541ecd5bc4298f324097803a1f270fdbe691389cedfd9",
+        version = "cdf306364eff8f50fd6473b32a316cb90cc15a2f",
+        sha256 = "0ccf19f30f744fca635be5428f6460c14dfee19bfa0820c70e0fc9554f79c9b1",
```

Comment:
The release `WAMR-12-30-2021` contains the commit you have here. Can we continue to use the release version?

https://github.com/bytecodealliance/wasm-micro-runtime/releases/tag/WAMR-12-30-2021

Reply:
This *is* WAMR-12-30-2021, just using commit version instead of a tag.

Reply:
Done.